### DISCUSSION TITLE
Front-End Development

### DISCUSSION THREAD

**Discussion by ImRanKhan81m** [ImRanKhan81m/Finix-Template#1] [Q&A] [answered]
How JavaScript work?

**Comment by MK-Hasan-Mahdi** [comment, marked as answer]
The way JavaScript works is interesting. Inside a normal Web page you place some JavaScript code (See How Web Pages Work for details on Web pages). When the browser loads the page, the browser has a built-in interpreter that reads the JavaScript code it finds in the page and runs it.
